onnxruntime/test/testdata/optimizers/test_adam.py

```diff
@@ -0,0 +1,160 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# orttraining_test_ortmodule_api.py
+
+import numpy as np
+import torch
+
+# PyTorch model definitions for tests
+np.set_printoptions(precision=10)
+torch.set_printoptions(precision=10)
+
+class SingleParamterModule(torch.nn.Module):
+    def __init__(self, input_size, hidden_size):
+        super(SingleParamterModule, self).__init__()
+        self.fc1 = torch.nn.Linear(input_size, hidden_size, bias=False)
+
+    def forward(self, input1):
+        out = self.fc1(input1)
+        return out
+
+class MultipleParamtersModule(torch.nn.Module):
+    def __init__(self, input_size, hidden_size, num_classes):
+        super(MultipleParamtersModule, self).__init__()
+        self.fc1 = torch.nn.Linear(input_size, hidden_size)
+        self.relu = torch.nn.ReLU()
+        self.fc2 = torch.nn.Linear(hidden_size, num_classes)
+
+    def forward(self, input1):
+        out = self.fc1(input1)
+        out = self.relu(out)
+        out = self.fc2(out)
+        return out
+
+def GenerateAdamWTestData(seed, model_setup_func, data_func, train_step_count):
+    torch.manual_seed(seed)
+    pt_model = model_setup_func()
+    adamw_optimizer = torch.optim.AdamW(pt_model.parameters(), lr=1e-3)
+
+    def run_step(model, x, target):
```

Comment:
Define functions in the module name space. Optionally prefix with `_` to show they are private